src/savers/tvg/tvgTvgSaver.cpp

```diff
@@ -108,6 +108,10 @@ static bool _merge(Shape* from, Shape* to)
     if (r != r2 || g != g2 || b != b2 || a != a2) return false;
 
     if (fabs(from->strokeWidth() - to->strokeWidth()) > FLT_EPSILON) return false;
+
+    //OPTIMIZE: Yet we can't merge outlining shapes unless we can support merging shapes feature.
+    if (from->strokeWidth() > 0 || to->strokeWidth()) return false;
```

Comment:
you've already checked that from->strokeWidth() and  to->strokeWidth() are the same, so now it's enough to check whether one of them is > 0

Reply:
A = 0
B = 255

if (A > 0) return false;

Reply:
in l.100:
```
if (fabs(A-B) > 0) return false;
```
so A == B in l. 113

Reply:
it's trivial. each line doesn't need to depend on the other condition.

Reply:
I hope you remember that I mentioned you before this kind of topic.

Reply:
I'm not sure whether I got your point, but if you think that the code is more clear like that, than you can merge it. 
it solves the penrose stairs issue :)